projects/go-lib/src/lib/components/go-modal/go-modal.component.html

```diff
@@ -1,9 +1,12 @@
 <div class="go-modal" [ngClass]="goModalClasses()" (click)="closeModalContainer($event)">
-  <div class="go-modal__container" #goModalContainer>
-    <div class="go-modal__close" (click)="closeModal()">
-      <go-icon icon="close"></go-icon>
+  <div [ngClass]="{'go-modal__container': modalSize === 'lg', 'go-modal__container-xl': modalSize === 'xl'}" #goModalContainer>
+    <div class="go-modal__header">
+      <h3 class="go-heading-3">{{ modalTitle }}</h3>
```

Comment:
Currently this looks very weird, but Andy confirmed they are okay with the look until `go-heading-*` classes are adjusted to match requirements.